frontends/web/src/routes/account/send/send.tsx

```diff
@@ -51,7 +51,7 @@ interface SendProps {
     deviceIDs: string[];
 }
 
-interface ProposedAmount {
+export interface Fee {
```

Comment:
`Fee` is not the right name, as it is used below also for non-fee values. What about `Amount`?